tests/AutomationApp/packages.config

```diff
@@ -1,9 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="AsyncUsageAnalyzers" version="1.0.0-alpha003" targetFramework="net452" developmentDependency="true" />
-  <package id="Microsoft.CodeAnalysis.FxCopAnalyzers" version="2.6.2" targetFramework="net452" developmentDependency="true" />
-  <package id="Microsoft.CodeQuality.Analyzers" version="2.6.2" targetFramework="net452" developmentDependency="true" />
-  <package id="Microsoft.NetCore.Analyzers" version="2.6.2" targetFramework="net452" developmentDependency="true" />
-  <package id="Microsoft.NetFramework.Analyzers" version="2.6.2" targetFramework="net452" developmentDependency="true" />
-  <package id="Text.Analyzers" version="2.6.2" targetFramework="net452" developmentDependency="true" />
-</packages>
```

Comment:
Was getting a FileLoadException when trying to run the automation app. Found that it's known issue - dotnet/standard#481 -

Part of the workaround was to use PackageReferences instead of packages.config